src/main/webapp/scripts/map.js

```diff
@@ -154,31 +163,57 @@ function getColourMarker(category) {
 }
 
 /* Place Marker Where Map is Clicked On & Show Popup*/
-function placeMarkerAndPanTo(latLng, map) {
-  var marker = new google.maps.Marker({
+function placeMarkerAndPanTo(latLng) {
+  /* Make marker for the clicked location */
+  const marker = new google.maps.Marker({
     position: latLng,
     map: map,
     icon: greyIcon
   });
-
+  /* Recenter the Map */
   map.panTo(latLng);
-  togglePopup();
+  /* Show the popup box */
+  togglePopup(latLng);
+  /* Update the latitude and longitude values in the popup */
   var markerPosition = marker.getPosition();
   populateLocation(markerPosition);
 
   /* If marker is right-clicked, delete */
-  google.maps.event.addListener(marker, 'rightclick', function(event) {
-      marker.setMap(null);
-  });
+   google.maps.event.addListener(marker, 'rightclick', function(event) {
+       marker.setMap(null);
+   });
```

Comment:
```suggestion
  google.maps.event.addListener(marker, "rightclick", (e) =>
    marker.setMap(null);
  });
```
I think it might be nice to consistently use double quote or single quote (I see you mostly use double quote)

Reply:
Thanks!